lib/data/auth/repository/auth_repository_impl.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:injectable/injectable.dart';
+import 'package:truckbill/data/auth/datasource/apple_data_source.dart';
+import 'package:truckbill/data/auth/datasource/google_data_source.dart';
+import 'package:truckbill/domain/auth/repository/auth_repository.dart';
+
+@injectable
+class AuthRepositoryImpl implements AuthRepository {
+  final GoogleAuthDataSource _googleAuthDataSource;
+  final AppleAuthDataSource _appleAuthDataSource;
+
+  AuthRepositoryImpl(this._googleAuthDataSource, this._appleAuthDataSource);
```

Comment:
Konstruktor na samej górze a zmienię pod nim

```suggestion
  AuthRepositoryImpl(this._googleAuthDataSource, this._appleAuthDataSource);
  
    final GoogleAuthDataSource _googleAuthDataSource;
  final AppleAuthDataSource _appleAuthDataSource;
```